dashboard/src/components/LoginForm/LoginForm.test.tsx

```diff
@@ -25,31 +28,16 @@ const defaultProps = {
   authenticating: false,
   authenticationError: undefined,
   location: emptyLocation,
-  checkCookieAuthentication: jest.fn(),
+  checkCookieAuthentication: jest.fn().mockReturnValue({
+    then: jest.fn(f => f()),
+  }),
   oauthLoginURI: "",
   appVersion: "devel",
+  authProxySkipLoginPage: false,
 };
 
 const authenticationError = "it's a trap";
 
-describe("componentDidMount", () => {
-  it("calls checkCookieAuthentication when oauthLoginURI provided", () => {
-    const props = {
-      ...defaultProps,
-      oauthLoginURI: "/sign/in",
-    };
-    const checkCookieAuthentication = jest.fn();
-    mount(<LoginForm {...props} checkCookieAuthentication={checkCookieAuthentication} />);
-    expect(checkCookieAuthentication).toHaveBeenCalled();
-  });
-
-  it("does not call checkCookieAuthentication when oauthLoginURI not provided", () => {
-    const checkCookieAuthentication = jest.fn();
-    mount(<LoginForm {...defaultProps} checkCookieAuthentication={checkCookieAuthentication} />);
-    expect(checkCookieAuthentication).not.toHaveBeenCalled();
-  });
-});
```

Comment:
Why are these removed?

Reply:
Because this was already been covered in the tests under "oauth login form" (sorry I forgot to mention that).